lib/softoken/pkcs11c.c

```diff
@@ -8279,6 +8279,54 @@ NSC_DeriveKey(CK_SESSION_HANDLE hSession,
                                   sourceKey, key);
             break;
 
+        case CKM_SP800_108_COUNTER_KDF:
```

Comment:
If we want to save space in a cluttered function, we could make this:

```c
case CKM_SP800_108_COUNTER_KDF:
case CKM_SP800_108_FEEDBACK_KDF:
case CKM_SP800_108_DOUBLE_PIPELINE_KDF
    crv = kbkdf_Dispatch(...);
    break;
```

Or some such, instead of adding all these lines here. That'd also allow us to unify handling of the additional safety checks in a single, common place for these functions. Thoughts?